models/ht_connection.h

```diff
@@ -200,8 +197,8 @@ template < typename targetidentifierT >
 HTConnection< targetidentifierT >::HTConnection()
   : ConnectionBase()
   , weight_( 1.0 )
-  , tau_P_( 50.0 )
-  , delta_P_( 0.2 )
+  , tau_P_( 500.0 )
+  , delta_P_( 0.5 )
```

Comment:
The actual values of these two parameters are not given in the Hill-Tononi (2005) paper. The new values are the default values in the original Synthesis simulator code by Sean Hill. I sent a mail to NEST User on 22 September warning of changes to the Hill-Tononi model and asking for comments from anyone who may be affected, but have not received a single answer, so I think we are safe here.